cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,18 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io/#/login');
 
+    cy.get(':nth-child(1) > .form-control')
+      .type('qatest03@gmail.com');
+
+    cy.get(':nth-child(2) > .form-control')
+      .type('!Qatest123');
+
+    cy.get('.btn')
+      .click();
+
+    cy.get(':nth-child(4) > .nav-link')
+      .should('contain.text', 'qatest03');
   });
```

Comment:
please add tests to check that the header contains the text 'Sign In' and the link contains 'login'

Reply:
Done

Reply:
Well done!